maintainers/maintainer-list.nix

```diff
@@ -8659,6 +8659,10 @@
       fingerprint = "4BFF 0614 03A2 47F0 AA0B 4BC4 916D 8B67 2418 92AE";
     }];
   };
+  nbr = {
+    email = "nbr@users.noreply.github.com";
+    name = "Nick Braga";
```

Comment:
need a github name and ID

Reply:
@jonringer Thanks for taking a look, I fixed this.